tests/test_routes.py

```diff
@@ -135,17 +129,6 @@ def test_create_shopcart(self):
         self.assertEqual(new_shopcart["customer_name"], test_shopcart.customer_name)
         self.assertEqual(new_shopcart["items"], test_shopcart.items)
 
```

Comment:
I think it would be beneficial to include the following code:

```
# Make sure server assigned a created/updated date
# (this will overwrite the ones from factory)
self.assertIsNotNone(test_shopcart.created_at)
self.assertIsNotNone(test_shopcart.last_updated)
```
This would ensure that created_at and last_updated are checked after retrieving the shopcart from the Location header, confirming that the server has assigned these timestamps properly. Let me know your thoughts!

Reply:
this is just a utility function to create using the factory so the server actually wont be assigning the dates in this case.